.github/renovate.json5

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Repository options (https://docs.renovatebot.com/configuration-options/)
+{
+  "extends": ["config:base"],
+
+  // This delays new PRs until after standup each day to reduce standup noise.
+  // Note that this does *not* delay updates to already-open PRs.
+  // Note also that this is implicitly limited by the trigger configuration
+  // in /.github/workflows/renovate.yml.
+  "timezone": "America/Los_Angeles", // Canonical IANA name for Pacific time
+  "schedule": ["after 12pm every weekday"],
+
+  // This single line is the primary reason we're using Renovate instead of
+  // Dependabot; it offers partial mitigation for some scenarios where a
+  // transitive dependency author's npm credentials are temporarily compromised,
+  // eg, https://github.com/faisalman/ua-parser-js/issues/536
+  "stabilityDays": 7,
+
+  // Note, this limits how many new PRs can be created per hour, *not* how many
+  // PRs may be open simultaneously
+  "prHourlyLimit": 2,
```

Comment:
I assume this is just for the onboarding phase?

Reply:
Yeah, I wanted to limit it while I was still testing. Will probably use ~5 in practice